bundles/core/src/main/java/com/adobe/cq/commerce/core/components/models/product/GiftCardAmount.java

```diff
@@ -0,0 +1,21 @@
+/*******************************************************************************
+ *
+ *    Copyright 2020 Adobe. All rights reserved.
+ *    This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License. You may obtain a copy
+ *    of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software distributed under
+ *    the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ *    OF ANY KIND, either express or implied. See the License for the specific language
+ *    governing permissions and limitations under the License.
+ *
+ ******************************************************************************/
+package com.adobe.cq.commerce.core.components.models.product;
+
+public interface GiftCardAmount {
```

Comment:
Is this really necessary? What is the uid used for? 

please add javadoc

Reply:
that was exposed keeping in mind if there would be any future requirement comes but yes we can do without it also.